src/assets/TokenDetectionController.ts

```diff
@@ -1,12 +1,18 @@
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
Why ignore this rule? It's asking you to use the named export instead. Not difficult to accommodate

Reply:
Fixed and updated in other parts of the code as well.